packages/ui/src/components/SearchInput/index.tsx

```diff
@@ -66,13 +67,16 @@ export const SearchInput = forwardRef(
       shortcut = false,
       error,
       disabled,
+      className,
     }: SearchInputProps,
     ref: Ref<HTMLInputElement>,
   ) => {
     const focusedLinkIndex = useRef(0)
     const popupRef = useRef<HTMLDivElement>(null)
     const [containerWidth, setContainerWidth] = useState(0)
     const [searchTerms, setSearchTerms] = useState('')
+    const [isMacOS, setIsMacOS] = useState(false)
```

Comment:
Do we need a react state for this ?

Reply:
@philibea yes we need as we cannot know if `window` exists or not depending on if the code is SSR or CSR